test/functional/connection/connection.ts

```diff
@@ -351,5 +353,19 @@ describe("Connection", () => {
         });
 
     });
+
+    describe("skip schema generation when skipSchemaSync option is used", function() {
+
+        let connections: Connection[];
+        beforeEach(() => setupTestingConnections({ entities: [View] }).then(all => connections = all));
+        afterEach(() => closeConnections(connections));
+        it("database should be empty after schema sync", () => Promise.all(connections.map(async connection => {
+            await connection.syncSchema(true);
+            const queryRunner = await connection.driver.createQueryRunner();
+            let schema = await queryRunner.loadSchemaTables(["view"], new DefaultNamingStrategy());
+            expect(!schema.some( table => table.name === "view" )).to.be.true;
```

Comment:
I think `!schema.some( table => table.name === "view" )` expression can be written in more human readable way (because I cant easily understand what this code does)